Spatial.gd

```diff
@@ -48,6 +48,20 @@ var taco_parser: TacoParser
 var x11_window_id_burrito: int
 var is_transient:bool = false
 
+# Scenes used throughout this scene
+var route_scene = load("res://Route.tscn")
+var icon_scene = load("res://Icon.tscn")
+var path2d_scene = load("res://Route2D.tscn")
+var gizmo_scene = load("res://Gizmo/PointEdit.tscn")
+
+##########Node Connections###########
+onready var marker_packs := $Control/Dialogs/MarkerPacks/MarkerPacks as Tree
+onready var root := self.marker_packs.create_item() as TreeItem
```

Comment:
You probably don't need to use .create_item() here because all it will do is create something that will be deleted.
```suggestion
onready var root: TreeItem
```